cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -453,6 +453,67 @@ TEST_F(RowBitCount, NestedTypes)
   }
 }
 
+TEST_F(RowBitCount, NullsInStringsList)
+{
+  using namespace cudf;
+  using namespace cudf::test;
+  using offsets_wrapper = fixed_width_column_wrapper<offset_type>;
+
+  // clang-format off
+  auto strings = std::vector<std::string>{ "daïs", "def", "", "z", "bananas", "warp", "", "zing" };
+  auto valids  = std::vector<bool>{            1,     0,   0,  1,         0,      1,   1,     1 };
+  // clang-format on
+
+  cudf::test::strings_column_wrapper col(strings.begin(), strings.end(), valids.begin());
+
+  auto offsets   = cudf::test::fixed_width_column_wrapper<int>{0, 2, 4, 6, 8};
+  auto lists_col = cudf::make_lists_column(
+    4,
+    offsets_wrapper{0, 2, 4, 6, 8}.release(),
+    strings_column_wrapper{strings.begin(), strings.end(), valids.begin()}.release(),
+    0,
+    {});
+  CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(row_bit_count(table_view{{lists_col->view()}})->view(),
```

Comment:
I would like to enforce consistency by either using `cudf::` everywhere, or not using it anywhere.